python/ray/tune/trainable.py

```diff
@@ -395,11 +403,45 @@ def stop(self):
         self._result_logger.close()
         self._stop()
 
+    @property
+    def logdir(self):
+        """Directory of the results and checkpoints for this Trainable.
+
+        Tune will automatically sync this folder with the driver if execution
+        is distributed.
+
+        Note that the current working directory will also be changed to this.
+
+        """
+        return self._logdir
+
+    @property
+    def iteration(self):
+        """Current training iteration.
+
+        This value is automatically incremented every time `train()` is called
+        and is automatically inserted into the training result dict.
+
+        """
+        return self._iteration
+
+    def get_config(self):
```

Comment:
To be consistent with the rest of the methods above, should could be, although I don't know if it's worth changing in this PR.
```
@property
def config(self):
        """Returns configuration passed in by Tune."""
        return self._config
```

Reply:
This ran into issues with RLlib here (`trainer.py`):

```python
        # Merge the supplied config with the class default
        merged_config = copy.deepcopy(self._default_config)
        merged_config = deep_update(merged_config, config,
                                    self._allow_unknown_configs,
                                    self._allow_unknown_subkeys)
        self.raw_user_config = config
        self.config = merged_config
        Trainer._validate_config(self.config)
        if self.config.get("log_level"):
            logging.getLogger("ray.rllib").setLevel(self.config["log_level"])
```
I'll merge first and we can see later if we need to fix (also @ericl doesn't like properties :) )